res/index.html

```diff
@@ -6,7 +6,7 @@
 <head>
 <meta charset="utf-8">
 <title>perf.html</title>
-<script src='analytics.js'></script>
+<script src='/analytics.js'></script>
```

Comment:
without this change, when on an URL like http://localhost:4242/public/d3fd7e63272e130fdc3ae54b861e8586f5a0d385/calltree/, this looks for http://localhost:4242/public/d3fd7e63272e130fdc3ae54b861e8586f5a0d385/calltree/analytics.js ;)